component/web/security/src/main/resources/conf/onBoarding_email_template.html

```diff
@@ -15,9 +15,9 @@ <h2 style="margin: 0;padding: 0; font-size: 16px;font-family: hevetica regular x
                     <td align="center" style="padding: 25px 0 25px">
                         <table width="90%" cellpadding="0" cellspacing="0" >
                             <tr>
-                                <td align="left" style="line-height: 18px;font-size: 14px;color:#333;font-family:hevetica regular x2;">
-                                    <p style="margin: 0 0 10px 0;line-height: 40px;font-size: 17px">&{onboarding.email.hello} ${USER_DISPLAY_NAME},</p>
-                                    <p style="margin: 0 0 10px 0;line-height:22px">&{onboarding.email.welcome} ${COMPANY_NAME}</p>
+                                <td align="left" style="line-height: 18px;font-size: 14px;color:#333;font-family: 'Helvetica', 'Arial', sans-serif;">
+                                    <p style="margin: 0 0 10px 0;line-height: 40px;font-size: 17px;text-transform: capitalize">&{onboarding.email.hi} ${USER_DISPLAY_NAME},</p>
+                                    <p style="margin: 0 0 10px 0;line-height:22px">&{onboarding.email.welcome} ${COMPANY_NAME}.</p>
```

Comment:
We will keep the same labels right now, no modification needed

Reply:
Done!